electron/electron.js

```diff
@@ -91,10 +91,33 @@ ipcMain.handle('save-output', async (event, savePath, extractedData) => {
       const outputFile = path.join(savePath, `mcode-extraction-patient-${i + 1}.json`);
       fs.writeFileSync(outputFile, JSON.stringify(bundle), 'utf8');
     });
-    // retuerning true indicates that the save process succeeded
+    // returning true indicates that the save process succeeded
     return true;
   } catch (error) {
     // return the error message
     return error.message;
   }
 });
+
+ipcMain.handle('save-config-as', async (event, schema) => {
+  const options = {
+    defaultPath: app.getPath('downloads'),
+    properties: ['createDirectory'],
+  };
+  return dialog
+    .showSaveDialog(null, options)
```

Comment:
I'm not sure if this is possible since I haven't looked at the show save dialog documentation, but do you know if it is possible to automatically append the `.json` file extension? Every file we download through the config editor will be a JSON file, and I think it could be nice if we default to that option, similar to how Word defaults to a `.docx` or Powerpoint defaults to `.pptx`.

This is what the dialog looks like for Word: 
![image](https://user-images.githubusercontent.com/30803904/126489022-e7b9785f-b264-4617-b227-de6218bbab77.png)

Reply:
I have made .json the default extension